packages/@aws-cdk/aws-ec2/lib/security-group.ts

```diff
@@ -288,6 +290,54 @@ export class SecurityGroup extends SecurityGroupBase {
     : new ImmutableImport(scope, id);
   }
 
+  /**
+   * Import an existing security group into this app.
+   */
+  public static fromLookup(scope: Construct, id: string, options: SecurityGroupLookupOptions): ISecurityGroup {
+    if (Token.isUnresolved(options.vpcId)
+      || Object.values(options.filter || {}).some(Token.isUnresolved)
+      || Object.keys(options.filter || {}).some(Token.isUnresolved)) {
+      throw new Error(`All arguments to SecurityGroup.fromLookup() must be concrete (no Tokens)`);
+    }
```

Comment:
I suspect that `Token.isUnresolved(options)` will do the trick without needing to recursive manually. Can you try?

Reply:
It will not, `isUnresolved` will only work for one level. We could make a `containsUnresolved()` function which is recursive though.